Bylaws.md

```diff
@@ -40,6 +40,12 @@ An antitrust compliance statement shall be read at the start of all Forum Meetin
     
  5. The allocation of customers, territories, or products in any way."
 
+At the start of a Forum, CWG or sub-group Teleconference, the following "Note Well" shall be read:
+
+"All participants are reminded that they must comply with the CA/Browser Forum anti-trust policy, code of conduct, and intellectual property rights agreement, which can be found in section 1.3 of the Bylaws and cabforum.org. Please contact the chair with any comments or concerns about these policies".
+
+The antitrust compliance statement or note well does not need to be re-read when it has been read during a larger-scope meeting. For example, if there are three consecutive meetings (Forum-level, Server Certificate Working Group, Validation Subcommittee of the Server Certificate Working Group), the "antitrust statement/note well" needs to be read only at the beginning of the Forum-level meeting. 
```

Comment:
The underlying assumption here would be that, in the case of consecutive meetings, the Forum-level meeting would always be the first? What should happen if another meeting starts the session of consecutive meetings?

Reply:
This is obviously not mandatory but changing the order means that the second meeting would have attendees that were not in the first meeting, and therefore the Chair of the second meeting would need to remind about the antitrust statement/note well so the new attendees are informed. Please let me know if this is too confusing or if you have a different proposal.

Reply:
I will close this out. We can reopen if the clarification is not enough.